src/common/src/util/ordered/serde.rs

```diff
@@ -31,7 +31,7 @@ use crate::types::{
 use crate::util::sort_util::{OrderPair, OrderType};
 use crate::util::value_encoding::serialize_cell;
 
-/// The special `cell_id` reserved for a whole null row is `i32::MIN`.
+/// The sentinel cell id is `-1_i32`, which is ensure to be the first kvpair in storage
```

Comment:
```suggestion
/// The sentinel cell id is `-1_i32`, which is ensured to be the first kv pair in the row.
```
By the way, it seems -1 will be the last one?